### ISSUE TITLE
[Bug][a11y]: Virtualized Table should include aria-rowcount attribute

### ISSUE THREAD

**Issue by majornista** [adobe/spectrum-web-components#3393]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

Table

### Expected behavior

When a Table is virtualized it should include an [`aria-rowcount`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-rowcount) attribute to communicate the number of total rows in the table, otherwise screen readers will announce only the number of rendered rows.

### Actual behavior

Virtualized table lacks `aria-rowcount` attribute, so screen readers will announce only the number of rendered rows rather than the total rows in the table.

### How can we reproduce this issue?

3. Scroll to '....'
4. Check console
5. See error
